pallets/permissions/src/benchmarking.rs

```diff
@@ -0,0 +1,45 @@
+use crate::*;
+
+use polymesh_primitives::{DispatchableName, PalletName};
+
+use frame_benchmarking::benchmarks;
+use sp_std::{iter, prelude::*};
+
+const MAX_PALLET_NAME: u32 = 512;
+const MAX_DISPATCHABLE_NAME: u32 = 1024;
+
+fn make_name(m: u32) -> Vec<u8> {
+    iter::repeat(b'x').take(m as usize).collect::<Vec<_>>()
+}
+
+benchmarks! {
+    _ {}
+
+    set_call_metadata {
+        let p in 1..MAX_PALLET_NAME;
+        let d in 1..MAX_DISPATCHABLE_NAME;
```

Comment:
My gut feeling is that the length of these names doesn't affect the weight.

Reply:
It might, just with a small constant factor.